jobs/delorean/jenkinsfiles/discovery/rc/github/Jenkinsfile

```diff
@@ -159,15 +119,12 @@ node {
             sh "git remote add intly ${GitHubUtils.ghTransformUrl(templatesForkGitUrl, 'ssh')}"
             sh "git fetch --all"
 
-            def productRegistryIds = getProductRegistryIDs(productName)
-            String productVersionId = getProductVersionID(productName)
-
             stage('Collect Product Images') {
                 dir(templatesDir) {
                     productImages = rhRegistries.collect { registry ->
-                        productRegistryIds.collect { registryID ->
+                        registryProjectIDs.collect { registryID ->
                             println "Checking for registry: ${registry}, registryID: ${registryID}"
-                            sh(returnStdout: true, script: "find . -name \'*.y*ml\' -exec grep -oP \'${registry}/${registryID}/[^[:blank:]]*${productVersionId}\' {} \\; | sort | uniq").split('\n')
+                            sh(returnStdout: true, script: "find . -name \'*.y*ml\' -exec grep -oP \'${registry}/${registryID}.*/[^[:blank:]].*\' {} \\; | sort | uniq").replace('"', '').split('\n')
```

Comment:
This no longer includes the tag (productVersionID) as an option, it was only being used by AMQ online. This will cause it to return more images (1 more), but since we check the image exists in the public registry anyway it doesn't really matter.